src/libs/Navigation/Navigation.js

```diff
@@ -35,6 +31,14 @@ function setDidTapNotification() {
     didTapNotificationBeforeReady = true;
 }
 
+/**
+ * Returns true if the Navigation is ready to navigate
+ * @returns {boolean}
+ */
+function isReady() {
+    return navigationRef.isReady();
+}
```

Comment:
**noting:** we have a usage of `navigationRef.isReady()` in the `canNavigate()` method. Looks like they're kind of doing the same thing - but one is logging if we aren't ready. Is that something we should use instead of creating a new method?